task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskDescriptionEditor.vue

```diff
@@ -28,24 +28,39 @@
       v-sanitized-html="value">
       {{ placeholder }}
     </div>
-    <exo-task-editor
-      v-if="displayEditor"
-      ref="richEditor"
-      v-model="inputVal"
-      :max-length="MESSAGE_MAX_LENGTH"
-      :id="task.id"
-      :placeholder="$t('task.placeholder').replace('{0}', MESSAGE_MAX_LENGTH)" />
+
+      <textarea
+        id="descriptionContent"
+        ref="editor"
+        v-model="value"
+        class="d-none"
+        cols="30"
+        rows="10"></textarea>
+
+      <div
+        v-if="displayEditor && editorReady"
+        :class="charsCount > MESSAGE_MAX_LENGTH ? 'tooManyChars' : ''"
+        class="activityCharsCount flex d-flex justify-end"
+        style="">
+        {{ charsCount }}/{{MESSAGE_MAX_LENGTH}}
+      <i class="uiIconMessageLength"></i>
+      </div>
+
+
+
+
```

Comment:
please execute `npm run lint` to format files

Reply:
I did It